docs/root/configuration/http_filters/cors_filter.rst

```diff
@@ -11,6 +11,31 @@ For the meaning of the headers please refer to the pages below.
 * :ref:`v2 API reference <envoy_api_msg_route.CorsPolicy>`
 * This filter should be configured with the name *envoy.cors*.
 
+.. _cors-runtime:
+
+Runtime
+-------
+
+The CORS filter supports the following runtime settings:
+
+cors.{runtime_key}.filter_enabled
+  The % of requests for which the filter is enabled. Default is 0.
+  If present, this will override the :ref: `enabled <envoy_api_msg_route.CorsPolicy>`
```

Comment:
:ref:\`enabled <envoy_api_field_route.CorsPolicy.enabled>\`

Reply:
enabled isn't a header so it can't be linked to

Reply:
It's kinda "hidden", but you can access it via https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/route/route.proto.html?highlight=shadow#envoy-api-field-route-corspolicy-enabled.

Reply:
ah very tricky. Thanks!